seahorse-workflow-executor/deeplang/src/main/scala/ai/deepsense/deeplang/DOperation.scala

```diff
@@ -98,9 +102,37 @@ abstract class DOperation extends Operation
   def inferGraphKnowledgeForInnerWorkflow(context: InferContext): GraphKnowledge = GraphKnowledge()
 
   def typeTag[T : ru.TypeTag]: ru.TypeTag[T] = ru.typeTag[T]
+
+  val reportType: ChoiceParam[ReportType] = ChoiceParam[ReportType](
+    name = "report type",
+    description = Some("Reports extensiveness"))
```

Comment:
Maybe something like:
"What kind of report should be generated"

Reply:
"What kind of report should be generated" looks like just synonym for report type.

Reply:
I feel that it adds new information. But I'm open for other ideas. You can have a look at other parameters descriptions in seahorse to get a feel of what we are aiming at.

Another idea:
have this param as boolean (yes/no) and have
* name: generate rich reports
* description: If this is enabled, extensive reports for output entities are generated. Note that this involve additional computation.

Reply:
We might introduce new report types and then old workflows would be incompatible.